src/sinks/elasticsearch/config.rs

```diff
@@ -184,6 +189,18 @@ impl BulkConfig {
     }
 }
 
+/// Distribution configuration.
+#[configurable_component]
+#[derive(Clone, Debug, Default)]
+#[serde(rename_all = "snake_case")]
+pub struct DistributionConfig {
+    /// Endpoints to which to distribute data.
+    pub endpoints: Vec<String>,
+
+    /// Time between reactivation attempts.
+    pub reactivate_delay_secs: Option<u64>,
```

Comment:
Can we call this like... `endpoint_retry_timeout_secs` or something? As well, add more information to the Cue documentation -- and can you actually sync the code comment of this field with the Cue documentation for it? -- to explain that this is related to when the endpoint becomes unhealthy and we stop sending traffic to it.